xtask/src/lib.rs

```diff
@@ -211,6 +211,14 @@ impl Package {
                 if config.contains("wifi") && config.contains("bt") {
                     features.push("coex".to_owned());
                 }
+                if config.contains("csi")
```

Comment:
I don't think there are config symbols for "csi", "esp-now", "sniffer" and "coex" 🤔